src/io/flutter/FlutterMessages.java

```diff
@@ -29,6 +29,16 @@ public static void showError(String title, String message) {
                        NotificationType.ERROR));
   }
 
+  public static void showMessage(String title, String message) {
```

Comment:
Maybe rename to showInfoToast and the previous method to showErrorToast. (Assuming that's what they do.)